whale

```diff
@@ -1,5 +1,7 @@
 docker run \
     -u "$(id -u):$(id -g)" \
     -v $(pwd):/work \
-    -w /work -it -p 3000:3000 \
+    -w /work -it \
+    -p 3000:3000 \ # React port
```

Comment:
@ArpitAgrawal001 Was this tested locally?
Retaining these comments wasn't working, right?